### PR TITLE
175 fix percentage precision 

### PR DESCRIPTION
**Pull request by kinspotting**
Closes https://github.com/api3dao/api3-dao/issues/175

### CODE REVIEW

**Comment by bbenligiray** [comment]
I had overwritten RewardUtils tests with StateUtils tests accidentally in https://github.com/api3dao/api3-dao/pull/208 so I restored those. I also pushed some commits that makes things a lot more readable imo. I recommend to review all commits individually.

**Review by bbenligiray — APPROVED** [review]
Would be nice if you guys did another review

File: packages/pool/test/StateUtils.sol.js [hunk 1]
```diff
@@ -606,7 +599,7 @@ describe("setAprUpdateStep", function () {
 describe("setProposalVotingPowerThreshold", function () {
   context("Caller is primary DAO Agent", function () {
     context(
-      "Proposal voting power threshold to be set is smaller than or equal to 10,000,000",
+      "Proposal voting power threshold to be set is smaller than or equal to 100%",
```
Comment (BritikovKI) [inline packages/pool/test/StateUtils.sol.js:602]: We have an inconsistency here and in the Line 576, if 10,000,000 is 100%, then 500,000 is 5%, not 0,5%

Comment (BritikovKI) [reply packages/pool/test/StateUtils.sol.js:602]: So we actually need to make `newAprUpdateStep = ONE_PERCENT.mul(5);`, but I think that is not actually that importatnt

Comment (BritikovKI) [reply packages/pool/test/StateUtils.sol.js:602]: P.S. the same goes for line 77, but I think this number is actually 10%, so we just need to fix names for context(from line 309)

**Review by BritikovKI — CHANGES_REQUESTED** [review]

File: packages/pool/test/StateUtils.sol.js (hunk 1 above)
Comment (bbenligiray) [reply packages/pool/test/StateUtils.sol.js:602]: Previously 1,000,000 was 1% and 100,000,000 was 100%
https://github.com/api3dao/api3-dao/blob/develop/packages/pool/contracts/StateUtils.sol#L72-L73
so the initial PR got the strings wrong, I fixed them.

Comment (bbenligiray) [reply packages/pool/test/StateUtils.sol.js:602]: 10,000,000 is actually 10%, I just forgot that the maximum proposal voting power threshold is 10% and not 100%. Pushed the fix.

**Review by BritikovKI — APPROVED** [review]
It is legitimate

**Pull request merged**